### PR TITLE
chore: add `editors/README.md`

### PR DESCRIPTION
**Pull request by rzvxa**

### CODE REVIEW

**Comment by rzvxa** [comment]
* **#4760** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/4760?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rzvxa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: editors/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ⚓ Oxc Editor Plugins
+
+## Official
+
+[VSCode](https://github.com/oxc-project/oxc/tree/main/editors/vscode)
+
+[(Neo)Vim](https://github.com/oxc-project/coc-oxc)
```
Comment (armano2) [inline editors/README.md:7]: https://github.com/oxc-project/oxc-intellij-plugin

Comment (rzvxa) [reply editors/README.md:7]: I don't use IntelliJ, Is that plugin in a working state? These are only working extensions.

Comment (Boshen) [reply editors/README.md:7]: That repos seems abandoned 😅

**Pull request merged**